generated/meson.build

```diff
@@ -0,0 +1,55 @@
+
+sourcery = executable('sourcery', join_paths(meson.source_root(), 'src', 'sourcery.cpp'), native: true)
```

Comment:
This could also be `'../src/sourcery.cpp'` I guess.

Reply:
Done that, since generated is included before src I can't define a files obj there, and didn't want to pollute the top-level meson.build with an extra variable for it.